ArduPlane/radio.cpp

```diff
@@ -285,7 +285,7 @@ void Plane::control_failsafe()
         }
     }
 
-    if(g.throttle_fs_enabled == 0) {
+    if (ThrFailsafe(g.throttle_fs_enabled.get()) != ThrFailsafe::Enabled) {
```

Comment:
I'd have preferred the two "no failsafe" options be nominated here rather than the !=.  But meh.